resources/migrations/001_update_migrations.yaml

```diff
@@ -6859,6 +6859,67 @@ databaseChangeLog:
             columnDataType: integer
       rollback: # we can't reliably add back the constraint while downgrading
 
+  - changeSet:
+      id: v50.2024-05-08T09:00:00
+      author: qnkhuat
+      comment: Add task_history.status
+      changes:
+        - addColumn:
+            tableName: task_history
+            columns:
+              - column:
+                  name: status
+                  type: varchar(21)
+                  remarks: "the status of task history, could be started, failed, success, unknown"
+                  constraints:
+                    nullable: false
+                  defaultValue: "unknown"
+
+  - changeSet:
+      id: v50.2024-05-08T09:00:01
+      author: qnkhuat
+      comment: Drop not null constraint for task_history.ended_at
+      changes:
+        - dropNotNullConstraint:
+            tableName: task_history
+            columnDataType: ${timestamp_type}
+            columnName: ended_at
+
+  - changeSet:
+      id: v50.2024-05-08T09:00:02
+      author: qnkhuat
+      comment: Drop not null constraint for task_history.duration
+      changes:
+        - dropNotNullConstraint:
+            tableName: task_history
+            columnDataType: int
+            columnName: duration
+
+  - changeSet:
+      id: v50.2024-05-08T09:00:03
+      author: qnkhuat
+      comment: Drop default value task_history.ended_at
+      changes:
+        - dropDefaultValue:
+            tableName: task_history
+            columnName: ended_at
+      rollback:
+        - addDefaultValue:
+            tableName: task_history
+            columnName: ended_at
+            defaultValueComputed: current_timestamp
+
+  - changeSet:
```

Comment:
we need this because for some reason MySQL requires a default value for ended_at